### DISCUSSION TITLE
Comparison with SMPlayer?

### DISCUSSION THREAD

**Discussion by ghost** [zaps166/QMPlay2#684] [General]
I have always used SMPlayer, but recently, Github suggested your project to me. I want to know the pros and cons of your project compared to SMPlayer. Thank you.

**Comment by zaps166** [comment]
Hi, I haven't used SMPlayer for many years (10?). I remember it's a front-end to external application which plays video (mplayer, mpv). QMPlay2 is a player all-in-one. Both mpv (SMPlayer's possible backend) and QMPlay2 are FFmpeg-based.

I can't tell pros and cons, because I don't know SMPlayer well :smile:

**Comment by zaps166** [comment]
Please look [here](https://github.com/zaps166/QMPlay2/discussions/686) as pros/cons for QMPlay2 (mpv is possible backend for SMPlayer) :smile:
